pages/nft/url/index.vue

```diff
@@ -287,8 +286,11 @@ export default class FetchIndex extends Vue {
       publisher,
       type = 'CreativeWork',
     } = this.iscnData
-    let { description = '' } = this.iscnData
+    let { description = '', keywords = '' } = this.iscnData
     description = this.truncate(description, 200)
+    if (!keywords.includes('NewsPress')) {
```

Comment:
```suggestion
    if (this.isNewsPress && !keywords.includes('NewsPress')) {
```

We still need to check this.isNewsPress?

Reply:
[🐛 Prevent duplicate keywords](https://github.com/likecoin/app-like-co/pull/488/commits/f4a88038960325042a9c758303d5158235a339e7)